wd_capabilities.au3

```diff
@@ -175,6 +175,8 @@ Func _WD_CapabilitiesAdd($key, $value1 = Default, $value2 = Default)
 
 	If $value1 = Default Then $value1 = ''
 	If $value2 = Default Then $value2 = ''
+	If IsBool($value1) Then $value1 = StringLower($value1)
+	If IsBool($value2) Then $value2 = StringLower($value2)
```

Comment:
This means, in case the "type" is a boolean, the StringLower() function will convert the `false` boolean to a `'false'` string and this does the magic, right? Interesting. Good catch @mlipok!